src/main/java/christmas/domain/discount/GiftDiscount.java

```diff
@@ -0,0 +1,20 @@
+package christmas.domain.discount;
+
+import christmas.domain.menu.Menu;
+import christmas.domain.Money;
+
+public class GiftDiscount {
+    public static final String NAME = "증정 이벤트";
+    private static final Money MIN_PRESENTATION_MONEY = new Money(120000);
```

Comment:
정말 사소한 가독성을 위한 팁입니다!
```suggestion
    private static final Money MIN_PRESENTATION_MONEY = new Money(120_000);
```